README.md

```diff
@@ -12,6 +12,18 @@ A fájlok **JSON** formátumúak, és az alábbi képpen néznek ki:
     "dmg": 14
 }
 ```
-**Feladat 2**
+**Feladat 2** \
 A program először leellenőrzi, hogy a parancssori argumentumok száma megfelelő-e *(Kettő fájl van-e megadva)*, majd, ha igen, akkor, annak a két fájlnak a létezését ellenőrzi le a **fileExists** nevű függvénnyel. Ha bármely fájl nem létezik a program kilép és kiírja melyik fájl az, amelyik nem létezik. Ha mindkét fájl létezik, akkor lefut a **battle** nevű függvény és kiírja, hogy *character_name* wins. Remaining HP: *xxx*.
 
+**Attack speed** \
+A **JSON** fájl kiegészül egy újabb sorral **attackCooldown** néven, amely felvehet nem egész értéket is.
+```json
+{
+    "name": "Béla",
+    "hp": 1500000,
+    "dmg": 14,
+    "attackCooldown": 10.7
+}
+```
+A **battle** nevű függvény, annyiban változott így, hogy a karakterek akkor támadják csak meg a másikat, ha az *attackCooldown* eléri a 0-t.
+
```

Comment:
Tedd bele a GH pages url-t légyszi.

Reply:
@vizvezetek Megcsináltam, amiket kértél, meg már a Battle sincs a main-ben. Így jó lesz a kód?
@hegyhati vagy @vizvezetek Az URL-link létrehozásában, illetve a Github Pages beállításában tudnál/tudnátok segíteni légyszíves?

Reply:
> ár a Battle sincs a main-ben. Így jó lesz a kód?
> @hegyhati vagy @vizvezetek Az URL-link létrehozásában, illetve a Github Pages beállításában tudnál/tudnátok segíteni légyszíves?

A keres ugye az volt, hogy erre legyen egy issue, hogy hova allitsam be, de ha megirod itt, az is jo nekem vegulis

Reply:
@hegyhati állítsd be a gh-pages branchre, a root-ra, légyszi :)

Reply:
Elmeletileg done.

Reply:
@hegyhati  köszi, működik.